### PR TITLE
add revoke doc

### LINKED ISSUE #1090: Document: the REVOKE command
### Related code PR

https://github.com/risingwavelabs/risingwave/pull/3363
https://github.com/risingwavelabs/risingwave/pull/3004
@yezizp2012 Please help provide more examples in terms of the supported usage.

### Which part(s) of the docs might be affected or should be updated? And how?

Examples:

```
REVOKE ALL PRIVILEGES ON ALL MATERIALIZED VIEWS IN SCHEMA schema1 FROM user1;
REVOKE ALL ON DATABASE db1 FROM user1;
REVOKE ALL PRIVILEGES ON ALL SOURCES IN SCHEMA schema1 FROM user1;
REVOKE GRANT OPTION FOR ALL ON DATABASE db1 from user1 GRANTED BY user;
REVOKE CREATE ON SCHEMA schema1 FROM user1;

```

### Reference

https://www.postgresql.org/docs/current/sql-revoke.html

### PR DESCRIPTION
**Pull request by hengm3467**
## Info

- **Description**

 Document REVOKE. The previous PR seems to be empty.

- **Notes**

  [ Include any supplementary context or references here. ]

- **Related code PR**

  [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves #1090

## For reviewers

- **Preview**

  [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **upcoming** version of the documentation. ]

- **Key points**

  [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/sql/commands/sql-grant.md [hunk 1]
```diff
@@ -5,70 +5,73 @@ description: Grant a user privileges.
...
 TO user_name [WITH GRANT OPTION] [GRANTED BY user_name];
 ```
 
-Grant a user source privileges.
+Grant source or table privileges to a user.
+
 ```sql
 GRANT {{SELECT | UPDATE | INSERT | DELETE} [, ...]| ALL [PRIVILEGES]} 
-ON {SOURCE source_name [, ...]
+ON { {SOURCE | TABLE} source_or_table_name [, ...]
```
Comment (yezizp2012) [inline docs/sql/commands/sql-grant.md]: After https://github.com/risingwavelabs/risingwave/pull/11725, `SOURCE` will only have `SELECT` privilege.

File: docs/sql/commands/sql-revoke.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+```sql
+REVOKE {CREATE | ALL [PRIVILEGES]} 
+ON SCHEMA schema_name [, ...]
+FROM user_name [GRANTED BY user_name];
+```
+
+Revoke source or table privileges from a user.
+
+```sql
+REVOKE {{SELECT | UPDATE | INSERT | DELETE} [, ...]| ALL [PRIVILEGES]}
```
Comment (yezizp2012) [inline docs/sql/commands/sql-revoke.md]: Ditto.

**Comment by hengm3467** [comment]
@yezizp2012 I updated the topics based on your comments. Please take a look again. Thanks.

**Review by yezizp2012 — APPROVED** [review]
LGTM!

**Pull request merged**